_posts/2021-07-19-changelog-151.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #151"
+date: 2021-07-19 12:00:00 +0300
+---
+
+
+## New Features
+
+* [#6392] Add `Unused import` inspection. Note, this inspection is currently disabled by default due to possible false-positive warnings.
```

Comment:
I'd add more details about current restrictions to clarify that the inspection can detect unused inspection in some cases only

Reply:
Are there any restrictions except for children modules (regarding https://github.com/intellij-rust/intellij-rust/issues/2158)?

Reply:
I don't remember other resstrictions
But @dima74 should know better

Reply:
Yes, child modules and false-positives are the only current restrictions

Reply:
Thanks!